manifests/nodejs.yml

```diff
@@ -936,12 +944,15 @@ tests/:
       Test_TelemetryInstallSignature: *ref_4_23_0
       Test_TelemetrySCAEnvVar: *ref_5_13_0
     test_trace_sampling.py:
-      Test_Trace_Sampling_Basic: *ref_5_16_0  #actual version unknown
-      Test_Trace_Sampling_Globs: *ref_5_16_0  #actual version unknown
+      Test_Trace_Sampling_Basic: *ref_5_16_0
+                                              # actual version unknown
```

Comment:
This is the only thing I dislike about `yamlfix` but unfortunately it's not configurable. The formatter does not properly handle formatting comments immediately after yaml references

Options here are to either
1. Deal with it as is - it only effects nodejs.yml
2. Update pre-formatted text to have the comment above the line, rather than after it
3. Not use `yamlfix`, but then there's no other Python YAML formatter so we would need to go back to `yamlfmt` which either requires go or a custom installation path

Reply:
cc @simon-id who put those refs in place

Reply:
we went for option 2